### PR TITLE
black: update to 21.5b2 and fix formatting issues (bug 1714229)

### PR DESCRIPTION
**Pull request by zzzeid**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ RUN addgroup --gid 10001 app \
 ENV HGPYTHON3=
 
 COPY requirements.txt /python_requirements.txt
+RUN pip install pip --upgrade
```
Comment (cgsheeh) [inline Dockerfile:43]: I think this needs to be `pip install --upgrade pip`

File: Dockerfile-dev [hunk 2]
```diff
@@ -28,6 +28,7 @@ ENV HGPYTHON3=
 
 # TODO: there are some packages here that we do not need.
 COPY requirements.txt /python_requirements.txt
+RUN pip install pip --upgrade
```
Comment (cgsheeh) [inline Dockerfile-dev:31]: `pip install --upgrade pip`

File: tests/test_landing_job.py [hunk 3]
```diff
@@ -90,7 +90,9 @@ def test_cancel_landing_job_fails_bad_input(db, client, landing_job, auth0_mock)
     )
 
     assert response.status_code == 400
-    assert response.json["detail"] == ("'IN_PROGRESS' is not one of ['CANCELLED']")
+    assert response.json["detail"] == (
```
Comment (cgsheeh) [inline tests/test_landing_job.py:93]: Is the addition of ` - 'status'` leftover from when this PR had Flask v2? Confused as to how this changes after this PR.

**Review by cgsheeh — CHANGES_REQUESTED** [review]

File: tests/test_landing_job.py (hunk 3 above)
Comment (zzzeid) [reply tests/test_landing_job.py:93]: This is because of the upgrade to connexion, not Flask.

**Pull request merged**